db/drivers/sqlite/db.c

```diff
@@ -184,11 +197,18 @@ int db__driver_create_database(dbHandle *handle)
         else {
             G_warning(_("The sqlite config option '%s' is not supported"),
                       "SQLITE_CONFIG_URI");
-            strcpy(name2, name);
+            len = G_strlcpy(name2, name, sizeof(name2));
+            if (len >= sizeof(name2)) {
```

Comment:
```suggestion
            if (G_strlcpy(name2, name, sizeof(name2)) >= sizeof(name2)) {
```

Reply:
Done

Reply:
Thanks, looks much better to me now.

The warning/error question remain though.